content/spire/getting-started.md

```diff
@@ -1,203 +1,353 @@
+To customize the behavior of the SPIRE Server and SPIRE Agent to meet your application’s needs you edit configuration files for the server and agent. 
 
-In `server.conf`, identify the `UpstreamCA "disk" { ... }` plugin configuration section, and modify `key_file_path` and `cert_file_path` appropriately:
+Note that the some of the configuration file default settings -- such as those for database choice for server data, key management backend, and upstream -- will work well for most evaluations. 
 
-```conf
+The following decisions influence how you set values in the configuration file: 
+
+* What you will name your server trust domain and your agent trust domain 
+
+Trust domain names must be identical in the server and the agent. 
+
+* Which node attestation method your application requires 
+
+This depends on your where your workload is running. Your choice of node attestation method determines which node-attestor plugins you configure SPIRE to use in Server Plugins and Agent Plugins sections of the SPIRE configuration files. You must configure at least one node attestor on the server and only one node attestor on the agent. 
+
+For simplicity’s sake, this guide demonstrates using the join token method for node attestation. 
+
+* Which workload attestation method your application requires. As with node attestation methods, your choice depends on the infrastructure your application’s workloads are deployed in (for example, SPIRE supports identifying workloads that run in Kubernetes).
+
+* Which type of database your application will use to store server data
+
+SPIRE employs a database to persist data related to workload identities and registration entries. By default, SPIRE bundles SQLite and sets it as the default for storage of server data. SPIRE currently also supports PostgreSQL. For production purposes, you should carefully consider which database to use. 
+
+* Which key management backend your application requires
+
+The key manager generates and persists the public-private key pair used for the agent SVID.  You must choose whether to store the private key on disk or in memory. For production purposes, you also might consider integrating a custom backend for storage purposes, such as a secret store.
+
+* Which trust root (“upstream certificate authority (CA)”) your application will use
+
+The SPIRE server provides a CA. If you’re, for example, using an external PKI system
+	that provides an upstream CA. you can configure SPIRE to use that instead.
+
+Once you’ve made these decisions, you can [configure the server](#step-4-configure-the-server) and [configure the agent](#step-5-configure-the-agent) accordingly, after [installing them](#step-3-install-the-server-and-agent). 
+
+## Step 2 Obtain the SPIRE Binaries
+
+Pre-built SPIRE releases can be found in the [GitHub SPIRE releases directory](https://github.com/spiffe/spire/releases). These releases contain both server and agent binaries.
```

Comment:
I think people should be directed to the [Downloads](https://spiffe.netlify.com/downloads/) page for SPIRE instead. That page provides a more convenient interface for fetching releases.